packages/node/src/protocol/utils/get-outcome-increments.ts

```diff
@@ -4,24 +4,51 @@ import { Contract } from "ethers";
 import { One, Zero } from "ethers/constants";
 import { BaseProvider } from "ethers/providers";
 import { BigNumber, bigNumberify, defaultAbiCoder } from "ethers/utils";
+import { isArray } from "util";
 
 import { StateChannel } from "../../models";
 
-function computeEthTransferIncrement(outcome): [string, BigNumber] {
-  const decoded = defaultAbiCoder.decode(["tuple(address,uint256)[]"], outcome);
-
-  if (
-    !(
-      decoded.length === 1 &&
-      decoded[0].length === 1 &&
-      decoded[0][0].length === 2
-    )
-  ) {
-    throw new Error("Outcome function returned unexpected shape");
+function unwrap(arr: any) {
```

Comment:
Why is the type `any`?

Reply:
because I want to do the `isArray` check at runtime